requirements.txt

```diff
@@ -1,3 +1,5 @@
 scipy
-pandas>=0.23
-numba>=0.44
+numpy>=1.16
+pandas==0.25.3
+pyarrow==0.15.1
+numba==0.48
```

Comment:
I'm not sure how it works. We need numba from specific channel. Can we somehow check it?

Reply:
The numba should be taken from intel channel.
I expect that we can deal with it using the right channel order in install command